ui/raidboss/timeline.ts

```diff
@@ -342,16 +352,45 @@ export class Timeline {
         durationEvents.push(event);
         continue;
       }
-      this.ui?.OnRemoveTimer(event, false, true);
+      this.ui?.OnRemoveTimer(event, true);
     }
+    // Do not clear keep alive events here, as this is part of a sync jump
+    // and keep alive timing is independent of timeline time.
 
     this.activeEvents = durationEvents;
   }
 
   private _RemoveExpiredTimers(fightNow: number): void {
     let activeEvent = this.activeEvents[0];
     while (this.activeEvents.length && activeEvent && activeEvent.time <= fightNow) {
-      this.ui?.OnRemoveTimer(activeEvent, true);
+      const event = activeEvent;
+      if (this.options.KeepExpiredTimerBarsForSeconds > 0) {
```

Comment:
(This introduces regression in the user experience where a previously valid negative KeepExpiredTimerBarsForSeconds value was set to the HTML5 minimum of [4ms](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#Specification). /s)